content/_index.md

```diff
@@ -4,6 +4,11 @@ description: Developing the Theory and Practice of Epidemiological Forecasting
 layout: landing
 
 carousel:
+  - pre: Delphi Careers
+    title: CMU Seeking an Executive Director for Delphi Research Group & Strategic Coordinator of Public Health Research Initiatives
+    ref: careers
+    alt: View Position
+    image: cmu-bldg-7-hero.jpg
   - pre: Delphi Careers
```

Comment:
We should nudge the general "We're Hiring!" carousel item into slot 3, otherwise we get two careers page items in a row